src/utils/flags/zyppflags.cc

```diff
@@ -365,10 +383,16 @@ int parseCLI( const int argc, char * const *argv, const std::vector<CommandGroup
 
             zypp::str::smatch what;
             if ( zypp::str::regex_match( scanned, what, rxexpr ) && what.size() >= 2 ) {
-              if( opt.name != what[1] )
-                ZYPP_THROW( UnknownFlagException( scanned ) );
+              if( opt.name != what[1] ) {
+                if ( onlyWarnOnAbbrevSwitches() ) {
+		  WAR << "Abbreviated commandline option will not be supported in future versions: --" << what[1] << ": use --" << opt.name << endl;
```

Comment:
Shouldn't we use the plural of option here:  "Abbreviated commandline options will not ..." , currently it could be understood that only the currently used one won't be supported anymore